.vscode/extensions.json

```diff
@@ -0,0 +1,28 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
```

Comment:
JSON doesn't support comments. Does that break VSCode: https://stackoverflow.com/questions/244777/can-comments-be-used-in-json

Reply:
This file was auto generated by VScode. VScode has support for comments using jsonc. See https://code.visualstudio.com/docs/languages/json#_json-with-comments